### PR TITLE
make lib more user friendly

### PR DESCRIPTION
**Pull request by gorangajic**
no need to include `DeviceEventEmitter` and listen on difficult to write events 😄

### CODE REVIEW

**Comment by evollu** [comment]
this is a good idea. Thanks.

File: index.js [hunk 1]
```diff
@@ -1,23 +1,39 @@
...
+FCM.requestPermissions = function requestPermissions() {
+  return FIRMessaging.requestPermissions();
+};
 
-}
+FCM.on = function on(event, callback) {
+  const nativeEvent = eventsMap[event];
+
+  const listener = DeviceEventEmitter.addListener(nativeEvent, (params) => {
+    const result = event === 'refresh' ? params.token : params;
```
Comment (evollu) [inline index.js]: can you keep it simple and always return params? I've updated readme

Comment (gorangajic) [inline index.js]: sure

**Comment by gorangajic** [comment]
commit updated

**Comment by evollu** [comment]
LGTM

**Comment by evollu** [comment]
@gorangajic there is a typo. I will fix it in my new feature code

**Comment by evollu** [comment]
@gorangajic I've made some changes to the code (refresh token event name and behavior). Can you take a look?

**Comment by gorangajic** [comment]
it's fine, thank you 😄

**Pull request merged**